RoddyCore/src/de/dkfz/roddy/config/converters/BashConverter.groovy

```diff
@@ -210,10 +211,15 @@ class BashConverter extends ConfigurationConverter {
         return listOfSortedValues
     }
 
-    private boolean isQuoted(String string) {
+    private static boolean isQuoted(String string) {
         (string.startsWith("'") && string.endsWith("'")) || (string.startsWith('"') && string.endsWith('"'))
     }
 
+
+    static String convertListToBashArray(List list) {
+        "(${RoddyIOHelperMethods.joinArray(list.toArray(), " ")})" as String
```

Comment:
It is Groovy code, you do not need the helper join method!

Also, are there whitespaces after ( and ) necessary?